src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -124,6 +124,9 @@ def transform_av_set_collection_output(av_sets):
 cli_command(__name__, 'vmss nic list-vm-nics', 'azure.mgmt.network.operations.network_interfaces_operations#NetworkInterfacesOperations.list_virtual_machine_scale_set_vm_network_interfaces', cf_ni)
 cli_command(__name__, 'vmss nic show', 'azure.mgmt.network.operations.network_interfaces_operations#NetworkInterfacesOperations.get_virtual_machine_scale_set_network_interface', cf_ni, exception_handler=empty_on_404)
 
+# VMSS Autoscale
+cli_command(__name__, 'vmss autoscale create', custom_path.format('create_vmss_autoscale'))
```

Comment:
Will the `create` become the only command under `autoscale`? If yes, suggest `enable-autoscale` or any other command names.